tee-worker/omni-executor/rpc-server/src/auth_utils.rs

```diff
@@ -43,6 +49,69 @@ pub fn verify_wildmeta_signature(
 	Ok(())
 }
 
+/// Verify WildmetaBackend signature against user operation hash
+pub fn verify_wildmeta_backend_signature(
+	signature: &str,
+	user_operations: &[SerializablePackedUserOperation],
+	chain_id: ChainId,
+	entry_point_address: Address,
+	expected_pubkey: &[u8; 33],
+) -> Result<(), ErrorObject<'static>> {
+	if user_operations.is_empty() {
+		error!("No user operations provided for signature verification");
+		return Err(ErrorObject::from(ErrorCode::ParseError));
+	}
+
+	// Decode signature from hex
+	let signature_bytes = decode_hex(signature).map_err(|e| {
+		error!("Failed to decode signature: {:?}", e);
+		ErrorObject::from(ErrorCode::ParseError)
+	})?;
+
+	if signature_bytes.len() != 65 {
+		error!("Invalid signature length: expected 65 bytes, got {}", signature_bytes.len());
+		return Err(ErrorObject::from(ErrorCode::ParseError));
+	}
+
+	let signature_array: [u8; 65] = signature_bytes
+		.try_into()
+		.map_err(|_| ErrorObject::from(ErrorCode::ParseError))?;
+
+	// Convert first user operation to PackedUserOperation for hashing
+	let first_user_op = convert_to_packed_user_op(user_operations[0].clone()).map_err(|e| {
```

Comment:
here only verify the first op may be not enough. After a short discussion with @Kailai-Wang , how about we verify all ops? Then we change `signature` -> `vector of signatures` ? one by one with exact match.